### ISSUE TITLE
[qt-declarative] CI build failure on x64-windows

### ISSUE THREAD

**Issue by BillyONeal** [microsoft/vcpkg#23824]
/cc @Neumann-A 

```
[1/2] "D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/qtdeclarative/x64-windows-dbg" "D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -S "D:/buildtrees/qtdeclarative/src/5453a29e5c-fe3994f387.clean" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=D:/downloads/tools/perl/5.32.1.1/perl/bin/perl.exe" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DQT_SYNCQT:PATH=D:/installed/x64-windows/tools/Qt6/bin/syncqt.pl" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DINSTALL_DOCDIR:STRING=../doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=../include/Qt6/" "-DINSTALL_TRANSLATIONSDIR:STRING=../translations/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=../share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=../share/Qt6/mkspecs" -G "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=D:/packages/qtdeclarative_x64-windows/debug"
FAILED: ../../x64-windows-dbg/CMakeCache.txt 
"D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/qtdeclarative/x64-windows-dbg" "D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -S "D:/buildtrees/qtdeclarative/src/5453a29e5c-fe3994f387.clean" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=D:/downloads/tools/perl/5.32.1.1/perl/bin/perl.exe" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DQT_SYNCQT:PATH=D:/installed/x64-windows/tools/Qt6/bin/syncqt.pl" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DINSTALL_DOCDIR:STRING=../doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=../include/Qt6/" "-DINSTALL_TRANSLATIONSDIR:STRING=../translations/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=../share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=../share/Qt6/mkspecs" -G "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=D:/packages/qtdeclarative_x64-windows/debug"
-- The CXX compiler identification is MSVC 19.31.31104.0
-- The C compiler identification is MSVC 19.31.31104.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE  
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE  
-- CMAKE_BUILD_TYPE was set to: 'Debug'
-- Check for feature set changes
-- Tool 'Qt6::qmltyperegistrar' will be built from source.
-- Could NOT find LTTngUST (missing: LTTNGUST_LIBRARIES LTTNGUST_INCLUDE_DIRS) 
Exit code 0xc0000409
```

I think this may have been missed somehow when we updated the build fleet to VS2022. I observe qtbase is already trying to ask cmake not to find this?

**Comment by Neumann-A** [comment]
duplicate.... see https://github.com/microsoft/vcpkg/issues/23746
fixed by #23755

**Comment by BillyONeal** [comment]
Thanks @Neumann-A
